modules/sagemaker/sagemaker-groundtruth/app.py

```diff
@@ -0,0 +1,144 @@
+from typing import Any, Optional
+import aws_cdk as cdk
+from aws_cdk import (
+    Stack,
+)
+from settings import ApplicationSettings
+from constructs import Construct
+from cdk_nag import NagSuppressions, NagPackSuppression
+from cdk_nag import AwsSolutionsChecks
+from lib.stacks.init import LabelingInitStack as InitStack
+from lib.stacks.labeling_pipeline import LabelingPipelineStack
+from lib.stacks.statemachine_pipeline import ExecuteStateMachinePipeline
+
+app = cdk.App()
+
+
+class AppConfig(cdk.Stack):
```

Comment:
This class doesn't seem to be a construct, but just a container of settings? I think it can be omitted entirely. Check [one of the other `app.py`](https://github.com/awslabs/aiops-modules/blob/3fe2060256dd5e8ca37398613ab9ca87c7b433cf/modules/sagemaker/sagemaker-templates-service-catalog/app.py) files, it should be able to be used almost verbatim:

* The imports will need to refer to the proper stack.
* The nags will need to be updated.